.github/workflows/release_master_fontpatcher-zip.yaml

```diff
@@ -0,0 +1,45 @@
+name: Release font patcher from master
+
+on:
+  create:
+  push:
+    branches: master
+    paths:
+      - font-patcher
+      - src/glyphs/**
+      - src/archive-font-patcher-readme.me
+      - bin/scripts/archive-font-patcher.sh
+      - bin/scripts/name_parser
+      - .github/workflows/release_master_fontpatcher-zip.yaml
+  workflow_dispatch:
+  
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    
+    steps:
+    
+    - name: Grab the script and its dependencies from the repo
+      uses: snow-actions/sparse-checkout@v1.2.0
+      with:
+        patterns: |
+          font-patcher
+          src/glyphs
+          src/archive-font-patcher-readme.md
+          bin/scripts/archive-font-patcher.sh
+          bin/scripts/name_parser
+          
+    - run: chmod +x 
+        font-patcher
+        bin/scripts/archive-font-patcher.sh
+        bin/scripts/name_parser        
+    - run: bin/scripts/archive-font-patcher.sh
+    - run: cp archives/FontPatcher.zip . -f
+    - uses: EndBug/add-and-commit@v9
+      with:
+        fetch: false
+        add: "FontPatcher.zip"
+        message: "[ci] update FontPatcher.zip"
```

Comment:
We always start capitalized (i.e. `s/update/Update/`), but I can also change that.